src/components/LanguageSwitching/LanguageSwitch.js

```diff
@@ -0,0 +1,5 @@
+import i18n from 'i18next';
+
+export const changeLanguage = (lng) => {
+  i18n.changeLanguage(lng);
+};
```

Comment:
this is a vanilla js function, not a js component, please move this file, and other translation files to src/context/languageContext/yourFile,js

also, decide on one name for the function and file, whilst the main thing you implement should be languagueContext, this file could be named switchLanguage.js, and function switchLanguage

Reply:
To raczej nie jest nic związanego z contextem w react, to zwykła funkcja która ma przyspieszyć zmianę języka, przynajmiej ja tą funkcje tak postrzegam. @kamil-ciesla

Reply:
zgadzam się, w moim początkowym zamyśle na tego taska było użycie react context, jednak okazało się to niepotrzebne, więc faktycznie context nie jest potrzebny